lib/decryptor.rb

```diff
@@ -5,12 +5,11 @@
 class Decryptor < Cryptor
 
   attr_reader :decrypt_key, :decrypt_offset, :decrypt_shift
-  attr_accessor :shift
 
-  def initialize(message, key, date = '') 
-    super(message, key, date) 
+  def initialize(message, key, date = '')
+    super(message)
     @decrypt_key = Key.new(key)
-    @decrypt_offset = (date == '' ? Offset.new : Offset.new(date))
```

Comment:
`decrypt_offset` is made more concise by changing the `initialize` method of the `Offset` class